arangod/RestHandler/RestBatchHandler.cpp

```diff
@@ -66,214 +67,240 @@ RestStatus RestBatchHandler::executeVst() {
   return RestStatus::DONE;
 }
 
-RestStatus RestBatchHandler::executeHttp() {
+void RestBatchHandler::processSubHandlerResult(std::shared_ptr<RestHandler> handler)
```

Comment:
The `shared_ptr` is copied here, but there is no need to copy it. I suggest to change the signature to use a const reference instead.